book.go

```diff
@@ -260,7 +260,7 @@ func (bk *book) parseRunner(k string, v any) error {
 func (bk *book) parseHTTPRunnerWithDetailed(name string, b []byte) (bool, error) {
 	c := &httpRunnerConfig{}
 	if err := yaml.Unmarshal(b, c); err != nil {
-		return false, nil
+		return false, err
```

Comment:
This place should not be err.
Look at the part where you call `parse*RunnerWithDetailed`

Reply:
Runner is determined by whether or not Unmarshal can be done.